.circleci/config.yml

```diff
@@ -128,7 +128,7 @@ jobs:
           name: Run sample tests.
           command: npm run samples-test
           environment:
-            GCLOUD_PROJECT: long-door-651
+            GCLOUD_PROJECT: node-gcloud-ci
```

Comment:
that's weird - why is this different?

Reply:
Firestore uses a different project than the rest of the nodejs- libraries. Apparently it can't be used in the same project that uses Datastore?

Reply:
Oh yeah!  That's a good reason.